src/main/groovy/com/github/benmanes/gradle/versions/updates/DependencyUpdatesReporter.groovy

```diff
@@ -19,13 +19,7 @@ import com.github.benmanes.gradle.versions.reporter.JsonReporter
 import com.github.benmanes.gradle.versions.reporter.PlainTextReporter
 import com.github.benmanes.gradle.versions.reporter.Reporter
 import com.github.benmanes.gradle.versions.reporter.XmlReporter
-import com.github.benmanes.gradle.versions.reporter.result.DependenciesGroup
```

Comment:
I don't know what the convention is here but I personally don't like star imports and since it has been changed I'd advise to revert this change.

Reply:
It seems that the company convention kicked in, will be reverted.